xbmc/interfaces/IAnnouncer.h

```diff
@@ -13,33 +13,35 @@
 class CVariant;
 namespace ANNOUNCEMENT
 {
-  enum AnnouncementFlag
-  {
-    Player        = 0x001,
-    Playlist      = 0x002,
-    GUI           = 0x004,
-    System        = 0x008,
-    VideoLibrary  = 0x010,
-    AudioLibrary  = 0x020,
-    Application   = 0x040,
-    Input         = 0x080,
-    PVR           = 0x100,
-    Other         = 0x200,
-    Info          = 0x400
-  };
+enum AnnouncementFlag
```

Comment:
Why no `enum class`?

Reply:
Definitely something to look up in the future but that will explode the diff (and is out of the scope of this PR too)

Reply:
So be it.